components/measurement/nettests/web_connectivity.js

```diff
@@ -49,53 +49,9 @@ const messages = defineMessages({
   },
 })
 
-export const checkAnomaly = ( testKeys ) => {
-  const {
-    accessible,
-    blocking,
-  } = testKeys
-
-  let anomaly = null
-  let hint = <FormattedMessage id='Measurement.Status.Hint.Websites.NoCensorship' />
-
-  if ((accessible === true || accessible === null) && blocking === null) {
-    hint = <FormattedMessage id='Measurement.Status.Hint.Websites.Error' />
-    if (accessible === true) {
-      anomaly = 'SITEUP'
-    } else if (accessible === null) {
-      anomaly = 'UNKNOWN'
-    }
-  } else if (accessible === false && (blocking === false || blocking === null)) {
-    anomaly = 'SITEDOWN'
-    hint = <FormattedMessage id='Measurement.Status.Hint.Websites.Unavailable' />
-  } else if (blocking !== null && blocking !== false) {
-    anomaly = 'CENSORSHIP'
-    hint = <FormattedMessage id='Measurement.Status.Hint.Websites.Censorship' />
-    // Further identify type of censorship
-    if (blocking === 'dns') {
-      anomaly = 'DNS'
-      hint = <FormattedMessage id='Measurement.Status.Hint.Websites.DNS' />
-    } else if (blocking === 'http-diff') {
-      anomaly = 'HTTPDIFF'
-      hint = <FormattedMessage id='Measurement.Status.Hint.Websites.HTTPdiff' />
-    } else if (blocking === 'http-failure') {
-      anomaly = 'HTTPFAILURE'
-      hint = <FormattedMessage id='Measurement.Status.Hint.Websites.HTTPfail' />
-    } else if (blocking === 'tcp-ip') {
-      anomaly = 'TCPIP'
-      hint = <FormattedMessage id='Measurement.Status.Hint.Websites.TCPBlock' />
-    }
-  }
-
-  return {
-    status: anomaly,
-    hint
-  }
-}
-
```

Comment:
I don't see this function call getting removed from this PR. Was this already dead code that is removed just now?

Reply:
Yes, it was already removed in an earlier PR.